config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,190 @@
+    helm lint ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from North America\!"
+
+    # install your chart
+    helm install ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from North America\!" --generate-name
+
+     # annotate service account
+    kubectl annotate sa -n autoneg-system default iam.gke.io/gcp-service-account=autoneg-system@[PROJECT_ID].iam.gserviceaccount.com
+
+    # ensure pods are ready
+    kubectl wait --for=condition=Ready pods --all
+
+    # check the service and ensure that `anthos.cft.dev/autoneg-status` annotation is present in the output
+    kubectl get svc node-app-backend -o=jsonpath='{.metadata.annotations}'
+    ```
+
+1. Repeat the step for the other cluster
+
+    ```bash
+    # switch the context to the second cluster. The command uses cluster name and zone based on the values used to create the clusters.
+    gcloud container clusters get-credentials cluster-eu --zone=europe-west2-a
+
+    # validate your chart
+    helm lint ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from Europe\!"
+
+
+    # install your chart
+    helm install ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from Europe\!" --generate-name
+
+     # annotate service account
+    kubectl annotate sa -n autoneg-system default iam.gke.io/gcp-service-account=autoneg-system@[PROJECT_ID].iam.gserviceaccount.com
+
+    # ensure pods are ready
+    kubectl wait --for=condition=Ready pods --all
+
+    # check the service and ensure that `anthos.cft.dev/autoneg-status` annotation is present in the output
+    kubectl get svc node-app-backend -o=jsonpath='{.metadata.annotations}'
+    ```
+
+1. Switch the context to the cluster that contains the configs for load balancing resources and run verify that multi-cluster ingress is configured
+
+    ```bash
+    # switch the context to the main cluster
+    gcloud container clusters get-credentials [CLUSTER NAME] --zone=[CLUSTER ZONE]
+
+    # if you created the load balancing resources in the namespace, other than default, switch the context to that namespace
+
+    # verify that your backend service has 2 backends attached (select index of "global" if prompted)
+    gcloud compute backend-services describe node-app-backend-service
+    ```
+
+    The backends section of the output should list both backends, for example:
+
+    ```yaml
+    backends:
+    - balancingMode: RATE
+      capacityScaler: 1.0
+      group: https://www.googleapis.com/compute/v1/projects/<project_id>/zones/us-central1-b/networkEndpointGroups/k8s1-37f1db7d-default-node-app-backend-80-486adca6
+      maxRatePerEndpoint: 100.0
+    - balancingMode: RATE
+      capacityScaler: 1.0
+      group: https://www.googleapis.com/compute/v1/projects/<project_id>/zones/europe-west2-a/networkEndpointGroups/k8s1-292a63d7-default-node-app-backend-80-636c84c5
+      maxRatePerEndpoint: 100.0
+      connectionDraining:
+      drainingTimeoutSec: 300
+    ```
```

Comment:
Thank you for adding the details! In the output of the command `gcloud compute backend-services describe node-app-backend-service` I saw no backend is attached.

Reply:
Had a debugging meeting offline. Now the solution works.